lib/nodes/proxyShapeBase.cpp

```diff
@@ -177,6 +178,20 @@ MayaUsdProxyShapeBase::initialize()
     retValue = addAttribute(excludePrimPathsAttr);
     CHECK_MSTATUS_AND_RETURN_IT(retValue);
 
+    loadPayloadsAttr = numericAttrFn.create(
+        "loadPayloads",
+        "lpl",
+        MFnNumericData::kBoolean,
+        0.0,
```

Comment:
Shouldn't this attribute be turned on by default? That would preserve existing behavior of loading payloads when opening the stage.

Reply:
let me fix that by doing in the definition line instead

Reply:
Oh, sorry! I read right past that `setDefault(true)` call.

But yeah, looks a bit cleaner to just have it in the initial definition. Thanks!